libs/apigateway-lambda-proxy/src/lib/metric-alarms.ts

```diff
@@ -0,0 +1,399 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as aws from '@pulumi/aws'
+
+const namespace = 'AWS/ApiGateway'
+
+/**
+ * Dimensions to identify an HTTP API in CloudWatch.
+ *
+ * The main difference is REST APIs are identified by name, while HTTP APIs are identified by id.
+ */
+export type HttpGateway = {
+    /**
+     * The API Gateway ID to monitor. Only valid for HTTP endpoints.
+     */
+    id: pulumi.Input<string>
+
+    /**
+     * The API name to monitor.
+     *
+     * For REST endpoints, this must be the name of the API created in AWS.
+     * For HTTP endpoints, this can be anything (it's only used in alerts).
+     */
+    name: pulumi.Input<string> | undefined
+
+    /**
+     * The stage to monitor
+     */
+    stage?: pulumi.Input<string>
+}
+
+/**
+ * Dimensions to identify a REST API in CloudWatch.
+ *
+ * The main difference is REST APIs are identified by name, while HTTP APIs are identified by id.
+ */
+export type RestGateway = {
```

Comment:
Prob can just remove the support for REST apis, we don't use them and the package these alarms are in doesn't use it?

Reply:
We could, but there's very little difference between REST alarms and HTTP alarms

Reply:
🤷🏻 removed